tests/MoBi.Tests/Presentation/Mapper/IndividualToParameterValuesMapperSpecs.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Linq;
+using FakeItEasy;
+using MoBi.Core.Extensions;
+using MoBi.Helpers;
+using MoBi.Presentation.Mappers;
+using MoBi.Presentation.Presenter;
+using OSPSuite.BDDHelper;
+using OSPSuite.BDDHelper.Extensions;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Core.Domain.Formulas;
+using OSPSuite.Core.Domain.Services;
+
+namespace MoBi.Presentation.Mapper
+{
+   public class concern_for_IndividualToParameterValuesMapper : ContextSpecification<IndividualToParameterValuesMapper>
+   {
+      private IObjectBaseFactory _objectFactory;
+      private ICloneManagerForModel _cloneManager;
+      private PathAndValueEntityToParameterValueMapper _expressionParameterToParameterValueMapper;
+
+      protected override void Context()
+      {
+         _objectFactory = A.Fake<IObjectBaseFactory>();
+         A.CallTo(() => _objectFactory.Create<ParameterValue>()).ReturnsLazily(x => new ParameterValue());
+         A.CallTo(() => _objectFactory.Create<ParameterValuesBuildingBlock>()).ReturnsLazily(x => new ParameterValuesBuildingBlock());
+         A.CallTo(() => _objectFactory.CreateObjectBaseFrom(A<IFormula>._)).ReturnsLazily(x => new ExplicitFormula().WithId(Guid.NewGuid().ToString()));
+
+         _cloneManager = new CloneManagerForModel(_objectFactory, new DataRepositoryTask(), A.Fake<IModelFinalizer>());
+
+         _expressionParameterToParameterValueMapper = new PathAndValueEntityToParameterValueMapper(_objectFactory, _cloneManager);
```

Comment:
Rename any reference to `expression` in this file because this test comes from the identical one targeting expression building block. But now, we are using this mapper to map IndividualParameter.

Reply:
Check this whole file.